server/src/main/java/org/elasticsearch/cluster/metadata/MetadataIndexTemplateService.java

```diff
@@ -1379,6 +1400,40 @@ static List<Map<String, AliasMetadata>> resolveAliases(
         return Collections.unmodifiableList(aliases);
     }
 
+    /**
+     * Resolve the given v2 template into a collected {@link Settings} object
```

Comment:
Is this ... true? Not sure what the Settings object refers to?

Reply:
Copy paste error.... I will fix it